task6.rb

```diff
@@ -22,10 +22,31 @@ def *(other_pint)
   end
 end
 
-pint1 = PseudoInt.new(ARGV[0].to_i)
-pint2 = PseudoInt.new(ARGV[1].to_i)
+def is_i?(num)
+  num =~ /\A[-+]?[0-9]+\z/
+end
+
+if is_i?(ARGV[0])
+  pint1 = PseudoInt.new(ARGV[0].to_i)
+else
+  puts "Error: #{ARGV[0]} is not a number"
+  return
+end
+
+if is_i?(ARGV[1])
+  pint2 = PseudoInt.new(ARGV[1].to_i)
+else
+  puts "Error: #{ARGV[1]} is not a number"
+  return
+end
 
-p pint1 + pint2
-p pint1 - pint2
-p pint1 / pint2
-p pint1 * pint2
\ No newline at end of file
+case ARGV[2]
```

Comment:
what will happen if we pass this argument that is not defined in when section? For example 'random string'

Reply:
Nothing?

Reply:
True. But your user won`t understand what happened. Please add else clause to your case statement.